Formula/ruby.rb

```diff
@@ -32,6 +33,23 @@ class Ruby < Formula
   depends_on "libyaml"
   depends_on "openssl"
 
+  resource "rubygems" do
+    url "https://rubygems.org/rubygems/rubygems-2.6.14.tgz"
+    sha256 "406a45d258707f52241843e9c7902bbdcf00e7edc3e88cdb79c46659b47851ec"
+  end
+
+  def program_suffix
+    build.with?("suffix") ? "24" : ""
+  end
+
+  def ruby
+    "#{bin}/ruby#{program_suffix}"
+  end
+
+  def abi_version
```

Comment:
Shouldn't this be `api_version`?

Reply:
Umm. I _think_ so, but considerably more intelligent & experienced devs than me have touched this code quite a bit over the years & left it as-is so I'm reluctant to disagree with them to an extent 🙈.

Reply:
`#{ruby} -e 'print Gem.ruby_api_version'` 🤔

Reply:
Yeah, I got the logic. Just... find it amusingly difficult to believe we've been writing it this way for years and... nobody has noticed? 🙈. I mean it's possible, but 😅.